Drivers/arm64/README.md

```diff
@@ -0,0 +1,17 @@
+<!--
+SPDX-FileCopyrightText: 2020 Frans van Dorsselaer
+SPDX-FileCopyrightText: 2025 Tyler Baker
+
+SPDX-License-Identifier: GPL-3.0-only
+-->
+
+# Declaration of License Compliance
+
+These drivers are from a binary distribution of VirtualBox, version 7.1.97, provided
+by Oracle Corporation, subject to the terms and conditions of the
+[GNU General Public License, version 3](https://www.gnu.org/licenses/gpl-3.0.html).
+
+With the binary distribution came a written offer to download the source code from
+<https://www.virtualbox.org/wiki/Testbuilds/>.
```

Comment:
If I'm not mistaken, that page only contains the binary builds. The normal source code link (https://download.virtualbox.org/virtualbox/) does not contain Test/Development code. Klaus mentioned it is in "trunk", but is that publicly available? The regular "trunk" SVN is for 7.1.6 (https://www.virtualbox.org/browser/vbox/trunk/). Maybe Oracle is violating their own GPL license ;)